### PR TITLE
(WIP) (Docs) Adding transformed markdown docs

### PR DESCRIPTION
**Pull request by hestonhoffman**
Here are the transformed docs. Michelle has offered to set up the ditamap, so don't merge this until she's finished that.

### CODE REVIEW

File: documentation/bolt_new_features.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
+---
+author: Jean Bond <jean@puppet.com\>
+---
+
+# New features
+
+New features added to Bolt in the 1.x release series.
```
Comment (adreyer) [inline documentation/bolt_new_features.md]: Is this page accessible? I think it was probably orphaned when we switched to a changelog

**Comment by adreyer** [comment]
github doesn't seem to want to compare these to the predocs. It would probably be best to commit them to predocs first so we can more easily compare for factual inaccuracies.

File: pre-docs/bolt_configure_orchestrator.md [hunk 2]
```diff
@@ -1,87 +1,8 @@
...
 
-If you're a Puppet Enterprise (PE) customer, you can configure Bolt to use the PE orchestrator and perform actions on managed nodes. Pairing PE with Bolt enables role-based access control, logging, and visual reports in the PE console.
+If you're a Puppet Enterprise \(PE\) customer, you can configure Bolt to use the PE orchestrator and perform actions on managed nodes. Pairing PE with Bolt enables role-based access control, logging, and visual reports in the PE console.
 
-For details about using Bolt with PE, see the [PE orchestrator documentation](https://puppet.com/docs/pe/latest/bolt_configure_orchestrator.html).
+For details about using Bolt with PE, see the [PE orchestrator documentation](https://puppet.com/docs/pe/latest/bolt_configure_orchestrator.html).
 
-<!--The remaining content on this page is used in the *PE* docs at bolt_configure_orchestrator.html-->
-# Using Bolt with orchestrator
+**Parent topic:**[Configuring Bolt](configuring_bolt.md)
```
Comment (adreyer) [inline pre-docs/bolt_configure_orchestrator.md]: What does this do in the generated docs?

File: pre-docs/bolt_deprecations_and_removals.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Deprecations and removals
```
Comment (adreyer) [inline pre-docs/bolt_deprecations_and_removals.md]: We can delete this in favor of the CHANGELOG right?

File: pre-docs/bolt_release_notes.md [hunk 4]
```diff
@@ -2,12 +2,12 @@
 
 New features, enhancements, known issues, and resolved issues for the Bolt 1.x release series.
 
-Version 1.2.0 released October 2018.
```
Comment (adreyer) [inline pre-docs/bolt_release_notes.md]: This should be deleted in favor of CHANGELOG

File: pre-docs/bolt_release_notes_new.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+author: Jean Bond <jean@puppet.com\>
+---
+
+# Bolt release notes
```
Comment (adreyer) [inline pre-docs/bolt_release_notes_new.md]: This should be deleted in favor of CHANGELOG

File: pre-docs/bolt_resolved_issues.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
+# Resolved issues
```
Comment (adreyer) [inline pre-docs/bolt_resolved_issues.md]: This should be deleted in favor of CHANGELOG

File: pre-docs/bolt_new_features.md [hunk 7]
```diff
@@ -0,0 +1,434 @@
+# New features
```
Comment (adreyer) [inline pre-docs/bolt_new_features.md]: This should be dropped in favor of the CHANGELOG

File: pre-docs/markdown.properties [hunk 8]
```diff
@@ -0,0 +1,477 @@
...
+#                     [ yes ]*
+
+# args.input.dir = 
+#                     Specifies the base directory for your documentation project.
+
+# args.output.base = 
+#                     Specifies the name of the output file without file 
+#                     extension.
+
+args.rellinks = noparent
```
Comment (adreyer) [inline pre-docs/markdown.properties]: is the carriage return intentional? It looks like its on the other uncommented line to.

Comment (lucywyman) [reply pre-docs/markdown.properties]: I'm not sure - this was copied from https://github.com/puppetlabs/nebula-docs/blob/master/markdown.properties - we may not need the file at all, or may need to make changes to it.

Comment (hestonhoffman) [reply pre-docs/markdown.properties]: If you'd prefer not to use the file, you can run the build with a couple of extra arguments. `-Dargs.rellinks=noparent -Dargs.gen.task.lbl=YES`

**Pull request merged**